src/base-element.js

```diff
@@ -84,6 +86,17 @@ import {vsyncFor} from './vsync';
  * before buildCallback and it might be called multiple times including
  * after layoutCallback.
  *
+ * The prerenderCallback is always called before layoutCallback, however
+ * they both can have loading work in parallel and either can resovle
```

Comment:
Done. It took me like 10 seconds of staring to really tell these are two different words... :hamster: